shared/team-building/index.stories.tsx

```diff
@@ -763,6 +763,7 @@ const load = () => {
 
   Sb.storiesOf('Team-Building/Phone Search', module).add('Empty Phone Search', () => (
     <PhoneSearch
+      continueLabel="Continue"
```

Comment:
@cjb Shouldn't this warrant a storyshot update?

Reply:
@thebearjew It just means that the text didn't change from the previous storyshot which also had Continue as the label :)